packages/docs-nextra/pages/tutorials/quickStart/quickStart-1.mdx

```diff
@@ -15,8 +15,7 @@ This tutorial covers a sample project in 5 parts; each part is about 5-10 minute
 
 # Part 1: Creating a Doenet Account & Starting an Activity
 
-<DoenetML>
-{`
+<DoenetViewer source={`
```

Comment:
Is there a reason this isn't a triple blacktic?

Reply:
No, just being lazy as I think Virginia is going to delete this.